contracts/container/contract.go

```diff
@@ -238,6 +239,42 @@ func Update(script []byte, manifest []byte, data any) {
 	runtime.Log("container contract updated")
 }
 
+// PutObject registers successful object PUT operation and notifies about
+// it. metaInformation must be signed by container nodes according to
+// container's placement, see [VerifyPlacementSignatures]. metaInformation
+// must contain information about an object placed to a container that was
+// created with meta-on-chain feature enabled.
+func PutObject(metaInformation []byte, sigs [][]interop.Signature) {
+	metaMap := std.Deserialize(metaInformation).(map[string]any)
+	cID := metaMap["cid"].(interop.Hash256)
+	if len(cID) != interop.Hash256Len {
+		panic("incorrect container ID")
+	}
+
+	if storage.Get(storage.GetContext(), append([]byte{containersWithMetaPrefix}, cID...)) == nil {
+		panic("container does not support meta-on-chain")
+	}
+
+	if !VerifyPlacementSignatures(cID, metaInformation, sigs) {
+		panic("signature verification failed")
+	}
+
+	oid := metaMap["oid"].(interop.Hash256)
```

Comment:
Should be verified too.

Reply:
you mean length? agree it should be verified but it is not used for the contract so i checked only the essentials. if we want to check oid, i would say we need to check all the map somehow. but the issue says: "It does not need to process/store these events"

Reply:
> process/store

I mean creating any indexes/DB. But some sanity checks are better be performed.